Tilt Cloud Team: homepage and blog post (#527)
windmilleng/tilt.build: blog/_posts/2020-4-21-tilt-cloud.md

Tilt Cloud is a SaaS backend for Tilt that lets you Release, Support, and Measure the Tilt developer experience for your team. If you maintain a Tiltfile that's used by three to three thousand people, Tilt Cloud is for you. The unit of Tilt Cloud organization is a Team. By linking your Tiltfile with a Tilt Cloud Team using the `set_team` function, Tilt Cloud functions as a control plane for the Tilt instances running on each developer's laptop.

Tilt Cloud will help DevEx teams
* Release improvements fast and safe
* Support users when they encounter problems
* Measure the ground truth across all users

We're announcing Tilt Cloud early to work with a handful of design partners. Many of these features are prospective, and we're looking for help prioritizing these features. If you want to upgrade your Tilt experience and shape the direction of Tilt Cloud, [create a Team now](https://cloud.tilt.dev/team/new). We'll reach out to discuss how we can work together.

## DevEx teams need Infrastructure

> Writing the code is just one step to delivering a better experience to the user, whether you're an AppDev and the user is shopping on a smartphone or you're a DevEx and the user is writing code on a laptop. In our conversations with DevEx teams, we've found three areas of infrastructure needs:

Reviewer: ```suggestion
If you're an AppDev creating an ecommerce app, your user is a shopper on a smartphone. If you're a DevEx engineer, your user is a software developer. But writing code is just one step in the end user experience. In our conversations with DevEx teams, we've found three areas of infrastructure needs:
```

Release
Support
Measure

### Release

> When an AppDev runs into foreseeable trouble that wastes their time, they lose faith in the DevEx teams. We're building Tilt Cloud to give DevEx teams confidence about releasing new versions of Tilt.

Reviewer: ```suggestion
When an AppDev runs into preventable trouble that wastes their time, they lose faith in the DevEx teams. We're building Tilt Cloud to give DevEx teams confidence about releasing new versions of Tilt.
```

#### Team Version Visibility

> The first feature of Tilt Cloud is Team Version Visibility. When you open your Tilt Cloud Team page, you'll see the users who've used Tilt with your team recently, along with the version they're using.

Reviewer: ```suggestion
The first feature of Tilt Cloud is Team Version Visibility. Open your Tilt Cloud Team page (or [create a new one first](https://cloud.tilt.dev/team/new)), you'll see the users who've used Tilt with your team recently, along with the version they're using.
```

#### Team Version Management

> Weekly Tilt releases bring improvements, and occasionally bugs. You'll be able to test new releases of Tilt before they're suggested to every AppDev on your team. All good? Click to promote the new version to your team. Find a problem? Contact us and we'll get a bugfix out before it affects your team.

Reviewer: ```suggestion
Weekly Tilt releases bring improvements, and occasionally incompatibilities with your existing project. You'll be able to test new releases of Tilt before they're suggested to every AppDev on your team. All good? Click to promote the new version to your team. Find a problem? Contact us and we'll get it fixed before it affects your team.
```

> ### Support

Reviewer: ```suggestion
### Support (coming soon)
```

When an AppDev encounters a problem, you want to unblock them quickly. Tilt Cloud will be able to give you more info to help them.

> #### Team Snapshots

Reviewer: ```suggestion
#### Team Snapshots (coming soon)
```

> Being able to see a user's screen can make problems obvious. Team Snapshots will let you see the same view that flummoxed them. Team Snapshots will only be visible to members of your team, so you can share data securely.

Reviewer: ```suggestion
Being able to see a user's screen can make problems obvious. Team Snapshots will let you see the same view that flummoxed them. Team Snapshots will only be visible to members of your team, so you can share data securely. For now, check out [non-team snapshots](https://docs.tilt.dev/snapshots.html). 
```

Author: I think it might be confusing to have that and try to describe each; if it's okay I'm going to leave that out.

Reviewer: Sure. Resolving. No change.

#### User Triage Context
When a user says something like  "is deploying the Shopping Cart Service supposed to be this slow?" or "I can't start the backend", context is important. You might give different responses if you knew this was the user's first time trying Tilt versus if they're a long-time user. Tilt Cloud will let you quickly grab the context you need to triage User reports and requests.

### Measure
DevEx efforts often rely on anecdotes. Oftentimes the first sign there's a problem is when a senior engineer complains to a Director or VP, which then gets routed to the build team as a Code Red. Tilt Cloud will let you analyze the data so you can fix regressions proactively, drive adoption of new workflows, and claim credit for improvements. DevEx engineers can more easily demonstrate their impact when data tells the story.

## Getting Started with Tilt Cloud

> If you want to use and shape a tool that helps you manage your DevEx efforts more effectively, go to TK:link and create a team.

Reviewer: ```suggestion
If you want to use and shape a tool that helps you manage your DevEx efforts more effectively, start by [creating a team on Tilt Cloud](https://cloud.tilt.dev/team/new).
```
